src/lib.rs

```diff
@@ -562,12 +562,11 @@ pub fn run_for_slice<'a>(
     if config.fix {
         panic!("Use run_fixing_for_slice()");
     }
-    let instantiated_rules = config.get_instantiated_rules();
```

Comment:
Noticed that this was unnecessarily getting re-created every time regardless of whether a `PerConfigContext` got passed in or not